picturae_csv_create.py

```diff
@@ -51,12 +50,14 @@ def init_all_vars(self, date_string):
                      'collector_number', 'collecting_event_guid',
                      'collecting_event_id',
                      'determination_guid', 'collection_ob_id', 'collection_ob_guid',
-                     'name_id', 'author_sci', 'family', 'gen_spec_id', 'family_id', 'parent_author']
```

Comment:
imported records_dropped variable (which is records dropped by TNRS) from picturae_csv_create, in order to include it the the upload batch report